### DISCUSSION TITLE
Icon Request-eye-closed

### DISCUSSION THREAD

**Discussion by MaureenT** [FortAwesome/Font-Awesome#21110] [Icon Requests]
Consider a closed eye icon as an alternative to the eye with a slash and as an opposite to the open eye. fa-eye  Closed eye can be represented by a lower half circle with eyelashes on the bottom.

**Comment by MaureenT** [comment]
Our use case for this request is best seen on our webpage.  We are a vision science research lab and use accordion drop downs on a few pages.  Instead of directional arrows, I use an open and closed eye.  See attached for the closed eye image I use.  I use the fa-eye for the open eye and would love a fa-eye-closed to use in place of the image.  Here is the page: http://www.mcw.edu/AOIP/Research/Publications.htm
![eye-closed](https://cloud.githubusercontent.com/assets/16600148/13183766/0a7bbec2-d6fe-11e5-84ba-f636d73cb8ab.png)

**Reply by igorsantos07** [reply]
Sadly, not true anymore. The website nowadays uses +/-.

**Comment by Jacob-McIver** [comment]
+1

**Comment by igorsantos07** [comment]
Definitely a must-have nowadays. It's a common duo on many apps to show/hide sensitive information. The slashed eye doesn't convey exactly the same information as a closed eye would do.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
We've finally added this to FA: https://fontawesome.com/icons/eye-closed?f=classic&s=solid
